examples/hmm.py

```diff
@@ -627,14 +638,14 @@ def main(args):
     # note that since we removed unseen notes above (to make the problem a bit easier and for
     # numerical stability) this test loss may not be directly comparable to numbers
     # reported on this dataset elsewhere.
-    test_loss = elbo.loss(model, guide, sequences, lengths, args=args, include_prior=False)
+    test_loss = elbo.loss(model, guide, sequences, lengths, batch_size=sequences.shape[0], include_prior=False)
     logging.info('test loss = {}'.format(test_loss / num_observations))
 
     # We expect models with higher capacity to perform better,
     # but eventually overfit to the training set.
     capacity = sum(value.reshape(-1).size(0)
                    for value in pyro.get_param_store().values())
-    logging.info('{} capacity = {} parameters'.format(model.__name__, capacity))
+    logging.info('{} capacity = {} parameters'.format(model_name, capacity))
```

Comment:
nit: maybe change to `'model{} capacity ...'.format(args.model)` to avoid need to save `model_name` above?